common/changes/@microsoft/rush/sachinjoseph-lazyLoadLocalProjects_2020-08-19-18-36.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Load and validate local projects lazily to further improve Rush startup times.",
+      "type": "patch"
```

Comment:
@sachinjoseph Are you making these JSON files manually?  I thought that for a version-policy package like Rush, the `rush change` command should assign `"type": "none"`.  (This is a weird design, but it is the current design.)

Reply:
Sorry I thought I made an error and manually edited it to say "patch". Let me revert it.